server/pom.xml

```diff
@@ -39,6 +44,19 @@
           </archive>
         </configuration>
       </plugin>
+      <plugin>
```

Comment:
this plugin bundles all dependencies into a single .jar. makes running it easy from command line.